Resources/Prototypes/GameRules/events.yml

```diff
@@ -428,17 +428,17 @@
     # minimumPlayers: 20
   # - type: ImmovableRodRule
 
-- type: entity
-  noSpawn: true
-  parent: BaseGameRule
-  id: IonStorm
-  components:
-  - type: StationEvent
-    weight: 5
-    earliestStart: 20
-    reoccurrenceDelay: 60
-    startAnnouncement: station-event-ion-storm-start-announcement
-    startAudio:
-      path: /Audio/Announcements/ion_storm.ogg
-    duration: 1
-  - type: IonStormRule
+# - type: entity
+#   noSpawn: true
+#   parent: BaseGameRule
+#   id: IonStorm
+#   components:
+#   - type: StationEvent
+#     weight: 5
+#     earliestStart: 20
+#     reoccurrenceDelay: 60
+#     startAnnouncement: station-event-ion-storm-start-announcement
+#     startAudio:
+#       path: /Audio/Announcements/ion_storm.ogg
+#     duration: 1
+#   - type: IonStormRule
```

Comment:
While I think I might have liked this in a separate branch to reference later, it's also so easy to play with, it's not worth it. :+1:

Reply:
My thinking was that this is in-scope for this PR on account of the fact that distress signals essentially replace the vanilla events system entirely (most of them are just annoying and do not actually offer any interesting gameplay).

Reply:
That said, ion storm good.